client/components/add-new-site/menu-item.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from '@wordpress/components';
+import { Icon } from '@wordpress/icons';
+import clsx from 'clsx';
+import { TranslateResult } from 'i18n-calypso';
+import React from 'react';
+
+const ICON_SIZE = 32;
+
+type Props = {
+	icon: JSX.Element;
+	iconClassName?: string;
+	heading: string;
+	description: string | TranslateResult;
```

Comment:
Just `TranslateResult` should be enough, no?

```suggestion
	description: TranslateResult;
```

Reply:
Yes. Fixed it [here](https://github.com/Automattic/wp-calypso/pull/97499/commits/8365a989faeeda3c1d128f37813e2595c66f08de).